packages/typespec-ts/src/modular/helpers/classicalOperationHelpers.ts

```diff
@@ -9,13 +9,26 @@ import {
 import { Client, OperationGroup } from "../modularCodeModel.js";
 import { getClassicalLayerPrefix, getClientName } from "./namingHelpers.js";
 import { getOperationFunction } from "./operationHelpers.js";
+import { SdkContext } from "../../utils/interfaces.js";
 
+export function shouldPromoteSubId(
+  dpgContext: SdkContext,
+  operationGroup: OperationGroup
+) {
+  const hasSubId = operationGroup.operations.some((op) =>
```

Comment:
lets use the whole subscription word, I'd rather make it clear than short

Reply:
updated.